File: Examples/Features/Worksheets/AutoFit.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Worksheets
{
    public class AutoFit : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];

            // Auto fit column width of range 'A1'
            worksheet.Range["A1"].Value = "Grapecity Documents for Excel";
            worksheet.Range["A1"].Columns.AutoFit();

            //Auto fit row height of range 'B2'
            worksheet.Range["B2"].Value = "Grapecity";
            worksheet.Range["B2"].Font.Size = 20;
            worksheet.Range["B2"].Rows.AutoFit();

            //Auto fit column width and row height of range 'C3'
            worksheet.Range["C3"].Value = "Grapecity Documents for Excel";
            worksheet.Range["C3"].Font.Size = 32;
            worksheet.Range["C3"].AutoFit();
        }
    }
}

File: Examples.VB/Features/Worksheets/AutoFit.vb
Namespace Features.Worksheets
    Public Class AutoFit
        Inherits ExampleBase

        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)

            ' Auto fit column width of range 'A1'
            With worksheet.Range!A1
                .Value = "Grapecity Documents for Excel"
                .Columns.AutoFit()
            End With

            'Auto fit row height of range 'B2'
            With worksheet.Range!B2
                .Value = "Grapecity"
                .Font.Size = 20
                .Rows.AutoFit()
            End With

            'Auto fit column width and row height of range 'C3'
            With worksheet.Range!C3
                .Value = "Grapecity Documents for Excel"
                .Font.Size = 32
                .AutoFit()
            End With
        End Sub

        Public Overrides ReadOnly Property IsNew As Boolean
            Get
                Return True
            End Get
        End Property
    End Class
End Namespace
